samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml

```diff
@@ -48,6 +50,8 @@ kibana:
 #         nginx.ingress.kubernetes.io/rewrite-target: /elasticsearch
 #         nginx.ingress.kubernetes.io/backend-protocol: HTTPS
 #       enabled: true
+#       # ingressClass is not currently supported by the Open Distro helm chart
+#       # ingressClassName: nginx
 #       path: /elasticsearch
```

Comment:
The comments on lines 53 and 54 are redundant, they duplicate the existing comments on lines 43 and 44.

Reply:
fixed